staging/src/k8s.io/client-go/util/connrotation/connrotation.go

```diff
@@ -81,10 +82,27 @@ func (c *ConnectionTracker) CloseAll() {
 	c.mu.Unlock()
 
 	for conn := range conns {
-		conn.Close()
+		_ = conn.Close()
 	}
 }
 
+// CloseAllGraceful forcibly closes all tracked connections that have been marked by GetCertOrTLSHandshakeComplete.
```

Comment:
I think that forcibly and Graceful are antonyms, if I understand the issue correctly I think that this is CloseAllEstablished or CloseAllExceptInflight

Reply:
for Graceful I understand that you signal somehow the other side to tell "I'm closing", but this just close it abruptly

Reply:
I struggled with coming up with a decent name.  `CloseAllEstablished` sounds okay.  Will see if Jordan has any ideas.